modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -117,6 +117,9 @@ function ding_webtrekk_page_alter(&$page) {
       // Track the page of the search result.
       $params['p_s_Page'] = $search_result->getSearchRequest()->getPage();
 
+      // Trak the size of the search result.
+      $params['p_s_Size'] = $search_result->getNumCollections();
```

Comment:
[`getNumCollections()` is limited by paging](https://github.com/ding2/ding2/blob/96b6c0422177dd5c6987a963ce84cad26e8af262/modules/ting/src/Search/TingSearchResultInterface.php#L15-L33). It seems as if the right way to go would be to use `getNumTotalObjects()`? 

NB: I see that the pull request comment states that we want to return the size of the current search page and we already track the size of the search result in `OSSr`. If that is the case then please update the comment accordingly.

Reply:
I updated the comment.